### ISSUE TITLE
Colour contrast issue with brand colour [accessibility]

### ISSUE THREAD

**Issue by BNewing** [get-alex/alex#259]
### Subject of the issue

There are some issues with colour contrast that stem from the brand purple. Small white text on top of a background the purple brand colour has an insufficient colour contrast. Similarly, purple brand colour text on top of the cream website background also has an insufficient colour contrast.

### Your environment

High Sierra, Mac, using Firefox - but this is an issue with colours, which will be present in all browsers.

### Steps to reproduce

I detected this error by running [WAVE, a tool created by Webaim to help with accessibility testing](https://wave.webaim.org/extension/).

### Expected behaviour

The colour contrast should be at least 4.5:1. This is specified in the [WCAG success criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum).

### Actual behaviour

It doesn't meet accessibility colour contrast standards.
